### PR TITLE
updated README, CONTRIBUTING and plugin metadata (bumped version etc.) (#88)

### LINKED ISSUE #88: Release Plugin
Now that we have a functional and useful plugin we should release the plugin.

### PR DESCRIPTION
**Pull request by adietish**
fixes #88

### CODE REVIEW

**Comment by adietish** [comment]
@jeffmaury, @lstocchi plz review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Contributions are essential for keeping this plugin great.
+We try to keep it as easy as possible to contribute changes and we are
+open to suggestions for making it even easier.
+There are only a few guidelines that we need contributors to follow.
+
+## First Time Setup
+1. Install prerequisites:
+   * [Java Development Kit](https://adoptopenjdk.net/)
+2. Fork and clone the repository
+3. `cd intellij-tekton`
```
Comment (jeffmaury) [inline CONTRIBUTING.md:12]: Should be intellij-kubernetes

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Run the plugin locally
+
+1. From root folder, run the below command.
+    ```bash
+    $ ./gradlew runIde
+    ```
+
+
+2. Once the plugin is installed and reloaded, there will be a Tekton Tab in the Tool Windows list (left).
```
Comment (jeffmaury) [inline CONTRIBUTING.md:23]: Kubernetes tab

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### Kubernetes & OpenShift resource tree
+The plugin offers a tool window with a tree that displays all the resources that exist on a Kubernetes or OpenShift cluster.
+The tree displays all the contexts that exist in your kube config (~/.kube/config).
+The tree is dynamically updated when external tools like kubectl, oc, etc. change the kube config. 
+
+#### Current Context
+The tree displays resources that exist on the cluster that the current context points to. 
+The user can switch the current context and cluster by choosing any context that's listed and picking "Set as Current Cluster" in the context menu.
+OpenShift clusters are marked as such with a OpenShift icon instead of a Kubernetes icon.
```
Comment (lstocchi) [inline README.md:19]: with an OpenShift icon

File: README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ 
+#### Current Namespace/Project
+The tree displays resources that exist within the current namespace as specified in the kube config. 
+Resources that are not bound to a namespace/project are also listed.
```
Comment (lstocchi) [inline README.md:23]: Maybe these two sentences can be merged. `The tree displays resources that exist within the current namespace/project and that are not owned to any namespace/project.`

**Review by lstocchi — COMMENTED** [review]
I think I'm not the best person to review an english documentation lol :laughing:. I put some comment, based on my personal opinions or some syntax errors I spotted.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# IntelliJ Kubernetes
+
+## Overview
+
+JetBrains IntelliJ plugin for interacting with Kubernetes and OpenShift clusters.
+The plugin tries to mimic the vscode-kubernetes-tools extension that exists for vscode.
```
[the remark this answers is not in the document]
Comment (adietish) [reply README.md:6]: Agree in the the concept where we get as close as possible to the vscode extension but depart from it where we think we can provide improvements.
On the other hands we are not just equal in terms of functionalities but also very close in visual representation. I dont want to nitpick the wording though. What about
> The plugin provides functionalities and user experiences that are very close to the Kubernetes extension for vscode (https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools).
?

[the remark this answers is not in the document]
Comment (lstocchi) [reply README.md:6]: +1, I like this much more

**Review by adietish — COMMENTED** [review]
@lstocchi https://github.com/redhat-developer/intellij-kubernetes/pull/158#discussion_r562652796: What about
> The user may switch the current namespace with the context menu item "Use Namespace". 
In OpenShift clusters the tree also lists projects which behave in an equivalent manner.

**Comment by lstocchi** [comment]
> @lstocchi [#158 (comment)](https://github.com/redhat-developer/intellij-kubernetes/pull/158#discussion_r562652796): What about
> 
> > The user may switch the current namespace with the context menu item "Use Namespace".
> > In OpenShift clusters the tree also lists projects which behave in an equivalent manner.

Sounds good :+1:

File: README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+
+**NOTE:** This plugin is in Preview mode. The plugin support for Kubernetes or OpenShift clusters is strictly experimental - assumptions may break, commands and behavior may change!
+
+## Release notes
+See the change log.
+
+Contributing
+============
+This is an open source project open to anyone. This project welcomes contributions and suggestions!
```
[the remark this answers is not in the document]
Comment (adietish) [reply README.md:48]: I copied this text from intellij-tekton. I dont think that `opens` is correct. The project is `open`.

[the remark this answers is not in the document]
Comment (jeffmaury) [reply README.md:48]: Agreed, is is the verb in this sentence

**Review by jeffmaury — APPROVED** [review]

**Comment by lstocchi** [comment]
@adietish do you plan to add a badge for showing the plugin page inside the IJ marketplace? Otherwise it could be a good idea to add it somewhere

**Review by lstocchi — APPROVED** [review]

**Comment by adietish** [comment]
> @adietish do you plan to add a badge for showing the plugin page inside the IJ marketplace? Otherwise it could be a good idea to add it somewhere in the readme when you have the link

@lstocchi, @jeffmaury I saw the badges in tekton and like them. They point to the intellij marketplace and our jenkins instance. Both dont exist yet, so I was guessing that we can add them once we created the jenkins job and published?

**Pull request merged**